### DISCUSSION TITLE
spotify library without premium

### DISCUSSION THREAD

**Discussion by unlive74** [BimmerGestalt/AAIdrive#551] [General]
can't enable Spotify library browsing without premium subscription (mod installed).  Is viewing the library a premium feature or can something be fixed?

**Comment by hufman** [comment]
Viewing your library is not a Spotify Premium feature. A recent new version of Spotify (version 8.7) did hide the Library item from the Browse list (#549), but latest build has a fix for that, and browsing Spotify's collection and your Library is available even on a test phone with a free account.
However, the code to control the Spotify app and browse the library relies on a Spotify SDK component that refuses to control modded Spotify versions. You must install the official Spotify client to browse the library (and AAIdrive should show a hint about this).
